text/0068-feat-login-into-portable-account-with-metamask.md

```diff
@@ -0,0 +1,116 @@
+- FIP: 68
+- title: Login into Portable account with metamask 
+- author: Sabyasachi Patra, @asabya
+- status: draft
+- created: 2023-03-01
+
+# Summary
+Addressing web3 login workflow to portable account where users can retrieve their wallet in a decentralized manner from browser with metamask.
+
+# Context, motivation and guide level explanation
+
+The solution worked out here allows to log in to the user account from any browser into the same account described in [FIP-59](https://github.com/fairDataSociety/FIPs/blob/master/text/0059-portable-account.md) created with
+[fdp-create-account](https://github.com/fairDataSociety/fdp-create-account) project.
+
+# Reference-level explanation
+
+For the login to work with metamask, user should already have a portable account created with [fdp-create-account](https://github.com/fairDataSociety/fdp-create-account).
+
+As we know while creating a portable account users can provide a 12 word mnemonic phrase for the portable wallet or a 12 word mnemonic will be generated for them.
+
+This solution starts with importing the portable wallet into metamask. After that it is a two-step process

```

Comment:
if I already imported my key into Metamask, why would I retrieve its encrypted credentials?

Reply:
Metamask will not give me the seed right!? we need the credentials to get the encrypted seed.

Reply:
why would I need the encrypted seed? the wallet is already imported into my wallet handler with the 12 words that you mentioned

Reply:
We need the seed for pod addresses.

Reply:
but you already imported the HD wallet with the 12 recovery words from which you can derive any amount of pods.

Reply:
I dont think metamask has a way of importing account with mnemonic, So we use a private key to import the root private key. 

I'm not familiar with a way where we can generate another account from that imported wallet.